base/iterators.jl

```diff
@@ -240,6 +240,32 @@ end
 iteratorsize{I1,I2}(::Type{Zip{I1,I2}}) = zip_iteratorsize(iteratorsize(I1),iteratorsize(I2))
 iteratoreltype{I1,I2}(::Type{Zip{I1,I2}}) = and_iteratoreltype(iteratoreltype(I1),iteratoreltype(I2))
 
+# unzip
+```jldoctest
```

Comment:
Only the code should be within \`\`\`jldoctest ... \`\`\` (i.e. the parts with `julia>`)

Also, the complete docstring should be within """ ... """, take a look at the docstring for `zip` on the lines above in this file!

(Also note that the examples are outdated after you updated `unzip` to only work with `AbstractZipIterators`)